src/dftbp/timedep/linrespgrad.F90

```diff
@@ -2380,18 +2743,18 @@ subroutine getExcMulliken(iAtomStart, pc, s, dqex)
     tmp = sum(pc * s,dim=2)
     dqex(:) = 0.0_dp
     do iAt1 = 1, size(dqex)
-      dqex(iAt1) = sum(tmp(iAtomStart(iAt1):iAtomStart(iAt1 + 1) -1))
+      dqex(iAt1) = sum(tmp(denseDesc%iAtomStart(iAt1):denseDesc%iAtomStart(iAt1 + 1) -1))
     end do
 
   end subroutine getExcMulliken
 
 
   !> Calculation of force from derivatives of excitation energy
-  !! 1. we need the ground and excited Mulliken charges
-  !! 2. we need P,(T,Z),W, X + Y from linear response
-  !! 3. calculate dsmndr, dhmndr (dS/dR, dh/dR), dgabda (dGamma_{IAt1,IAt2}/dR_{IAt1}),
-  !! Dgext (dGamma-EXT_{IAt1,k}/dR_{IAt1})
-  subroutine addGradients(sym, nxov, natom, species0, iAtomStart, norb, homo, getIA,&
+  !> 1. we need the ground and excited Mulliken charges
```

Comment:
Second and further lines of routine comments are !! for doxygen/Ford. Most of the code it gradually changing to this: 
```suggestion
  !! 1. we need the ground and excited Mulliken charges
```